src/components/forms/FormItem.js

```diff
@@ -67,6 +67,26 @@ export default class FormItem extends React.PureComponent {
     return value.trim();
   }
 
+  renderErrorList(errors) {
+    return (
+      <ul className="error-list">
+        {Object.entries(errors).map(([k, v]) => {
+          if (typeof v === 'string') {
+            return <li key={k}>{`${k}: ${v}`}</li>;
+          } else if (v instanceof Array) {
+            return <li key={k}>{`${k}: ${v.join(' ')}`}</li>;
+          }
+
+          return (
+            <li key={k}>
+              {k} {this.renderErrorList(v)}
```

Comment:
This `k` isn't an English word so perhaps it should be:
```javascript
<li key={k}>
  <code>{k}</code> {this.renderErrorList(v)}
</li>
```

Reply:
good idea